src/classes/AFFL_MultiRecordType_TDTM.cls

```diff
@@ -74,8 +74,18 @@ public class AFFL_MultiRecordType_TDTM extends TDTM_Runnable {
 				for (SObject so : newlist) {
 					Affiliation__c affl = (Affiliation__c)so;
 
-					String lookupFieldLabel = afflMapper.mapAccRecTypeToContactLabel.get(affl.Affiliation_Type__c);
-					String lookupFieldName = afflMapper.contactLabelNames.get(lookupFieldLabel);
+					String lookupLabelOrAPINameFromMapping = afflMapper.mapAccRecTypeToContactLabel.get(affl.Affiliation_Type__c);
```

Comment:
I'd suggest removing "map" from "mapAccRecTypeToContactLabel". We try not to use Hungarian notation, as in a statically-typed language is superfluous and makes the variable names longer.

Reply:
I'd shorten "lookupLabelOrAPINameFromMapping" to just "lookupLabelOrAPIName". We don't really need to estate where it's coming from...

Reply:
Hi @ceiroa "mapAccRecTypeToContactLabel" was already there and looks like it is being used on AFFL_MultiRecordTypeMapper in a bunch of places, also it being public used in atleast 3 other classes multiple times changing it needs to be changed in all  those classes too...do we still want to change it?

Reply:
@ceiroa Updated the name of the map too, can you give it another shot? Thanks.